src/components/DataTable.tsx

```diff
@@ -77,6 +79,7 @@ export function DataTable(props: Props) {
   const queryClient = useQueryClient();
 
   const [filter, setFilter] = useState('');
+  const [customFilter, setCustomFilter] = useState<string[]>(['']);
```

Comment:
Does this need to have a default value of one empty string? Why can't we provide it with just `[]`?

Reply:
Sure, removed!